apps/package.json

```diff
@@ -49,7 +49,7 @@
     "@code-dot-org/js-interpreter-tyrant": "0.2.2",
     "@code-dot-org/js-numbers": "0.1.0-cdo.0",
     "@code-dot-org/p5.play": "1.1.9-cdo",
-    "@code-dot-org/piskel": "0.10.0-cdo.0",
+    "@code-dot-org/piskel": "0.11.0-cdo",
```

Comment:
Note: The release of Piskel that you've upgraded to [is `0.13.0`](https://github.com/code-dot-org/piskel/commit/2f2b4cd9bab6b44fab68e875133fabc28d35755e).  The version numbers in our fork were designed both to avoid conflicts with upstream versions, and to make it obvious which upstream version we're based on.  Here, since we've just merged version `0.13.0` into our fork, I'd expect the newly-released version to be `0.13.0-cdo.0`.  Later if we make changes to our fork and release a new version without new changes from upstream it would become `0.13.0-cdo.1`; if we merge upstream again it would become `<upstream-version>-cdo.0`.